tutorials/cloudsdk-ci-cd/cloudsdk-ci-cd.md

```diff
@@ -0,0 +1,252 @@
+
+![GitHub Enterprise Server](github-enterprise-server.png)
+
+### Create a Jenkins job for your project
+To create a build job for your project, navigate to `New Item` in the Jenkins main menu. On the following screen, choose `Multibranch Pipeline` and specify a name for your project´s build job.
+
+![Jenkins Create Build](jenkins_create_build.png)
+
+Next, you need to setup the SCM source of your project´s build job. Look for `Branch Sources` and add one by specifying your GitHub Enterprise endpoint, organization (via field `Owner`), and repository. If your repository is set to private, please also create and use a pair of suitable credentials and choose them for your repository. If you are using github.com, this will also protect you from running into rate limitations. To control when Jenkins will start a build run and what it will build during that run, you can use the `Behaviors` section of the configuration. The example below shows a configuration that will build all branches and GitHub Pull Requests. Finally, save your updated configuration by clicking the `Save` button on the bottom of the page.
+
+![Branche Source](branche-source.png)
+
+Congratulations! Jenkins will now scan your repository and trigger a build.
+
+The best way to get an overview of the build status is to open the job, navigate to the currently running build, and opening it in the blue ocean UI of Jenkins.
+
+![Jenkins Blue Ocean](jenkins-blue-ocean.png)
+
+![Pipeline](pipeline.png)
+
+Please note that your pipeline skipped the deployment stage because you did not configure a deployment target yet. To deploy your application to your SCP account, you need to add a target to your `pipeline_config.yml` file. Moreover, if you already built integration tests that require credentials, you also need to create corresponding records in the Jenkins credentials store and map them via an appropriate entry in the configuration file. Credentials can be created by navigating to `Credentials` » `System` » `Global credentials`.
+
+![Jenkins System](jenkins-system.png)
+
+On this screen click on `Add Credentials` and enter username, password, as well as a ID for your credentials record.
+
+![Add Credentials](jenkis-add-credentials.png)
+
+As a result, navigating to `Credentials` should show your freshly created entry:
+
+![Jenkins Credentials](jenkins-credentials.png)
+
+Finally, you can now leverage this credentials record by adding the credentials configuration property to the `integrationTests` stage of your `pipeline_config.yml`. The example below shows am example of mapping it to the system alias `MyErpSystem`, which you defined earlier in your `systems.yml` or `json` file.
```

Comment:
example is written twice and there is typo ("am") -- if possible, get tutorial copyedited.